brooklyn-tosca-transformer/src/test/java/io/cloudsoft/tosca/a4c/brooklyn/ToscaPlanToSpecTransformerTest.java

```diff
@@ -324,7 +324,7 @@ public void testDeploymentArtifacts() {
                         "brooklyn-example-hello-world-sql-webapp-0.6.0.war");
     }
 
-    @Test
+    //@Test failing to parse tosca
```

Comment:
Write `@Test(enabled = false)` instead. It will be reported in TestNG's output as skipped.